bundles/model/org.eclipse.smarthome.model.persistence/src/org/eclipse/smarthome/model/persistence/internal/PersistenceModelManager.java

```diff
@@ -67,7 +67,8 @@ protected void activate() {
     protected void deactivate() {
         modelRepository.removeModelRepositoryChangeListener(this);
         for (String modelName : modelRepository.getAllModelNamesOfType("persist")) {
-            manager.stopEventHandling(modelName);
+            String serviceName = modelName.substring(0, modelName.length() - ".persist".length());
```

Comment:
Should we use dbId instead of service name here?

Reply:
I used `serviceName` to stay consistent with `activate` where we already use `serviceName` for the same information, but I’m fine either way.